fvm/evm/emulator/state/collection.go

```diff
@@ -47,16 +47,22 @@ func NewCollectionProvider(
 // calling twice for the same collection might result in odd-behaviours
 // currently collection provider doesn't do any internal caching to protect against these cases
 func (cp *CollectionProvider) CollectionByID(collectionID []byte) (*Collection, error) {
-	storageID, err := atree.NewStorageIDFromRawBytes(collectionID)
+	slabID, err := atree.NewSlabIDFromRawBytes(collectionID)
 	if err != nil {
 		return nil, err
 	}
+
+	// TODO: expose SlabID.Address() in atree
```

Comment:
@fxamacker is this maybe available now?

Reply:
@turbolent Not yet, I just opened https://github.com/onflow/atree/issues/431 to add `SlabID.Address()`.

I can replace the workaround here after I add `SlabID.Address()` to atree.